packages/nextjs/components/blockexplorer/TransactionsTable.tsx

```diff
@@ -0,0 +1,130 @@
+import { ethers } from "ethers";
+import { TransactionHash } from "~~/components/blockexplorer/TransactionHash";
+import { Address } from "~~/components/scaffold-eth";
+import { getTargetNetwork } from "~~/utils/scaffold-eth";
+import { TransactionWithFunction, TransactionsTableProps } from "~~/utils/scaffold-eth/block";
+
+export const TransactionsTable = ({ blocks, transactionReceipts, isLoading }: TransactionsTableProps) => {
+  const targetNetwork = getTargetNetwork();
+
+  return (
+    <div className="overflow-x-auto shadow-lg">
+      <table className="min-w-full divide-y divide-primary shadow-lg rounded-lg bg-neutral">
```

Comment:
Check the DaisyUI table component:

https://daisyui.com/components/table/

For all the tables, we should rely on it all we can.

P.S. I like the hover effect on rows (also included on DaisyUI), but maybe only for the `TransactionsTable`. I don't see its value in other places, like in the [txHash] table